tslib.d.ts

```diff
@@ -34,3 +34,4 @@ export declare function __importStar<T>(mod: T): T;
 export declare function __importDefault<T>(mod: T): T | { default: T };
 export declare function __classPrivateFieldGet<T extends object, V>(receiver: T, privateMap: WeakMap<T, V>): V;
 export declare function __classPrivateFieldSet<T extends object, V>(receiver: T, privateMap: WeakMap<T, V>, value: V): V;
+export declare function __createBinding(t: any, mod: any, k: string, k2?: string): void;

```

Comment:
Could you add a **TSDoc** comment for this like I’m doing in <https://github.com/microsoft/tslib/pull/91>?

Also, this should probably be:
```suggestion
export declare function __createBinding(t: object, mod: object, k: string, k2?: string): void;
```

or even:
```suggestion
export declare function __createBinding(t: object, mod: object, k: PropertyKey, k2?: PropertyKey): void;
```